packages/datadog-plugin-graphql/src/resolve.js

```diff
@@ -80,6 +80,11 @@ class GraphQLResolvePlugin extends TracingPlugin {
     // this will disable resolve subscribers if `config.depth` is set to 0
     super.configure(config.depth === 0 ? false : config)
   }
+
+  finish (finishTime) {
+    this.activeSpan.finish(finishTime)
+    super.finish()
```

Comment:
Can you omit the `super.finish()`? It will just call `this.activeSpan.finish()` again.

Reply:
Yep, can do! I wasn't sure about the conventions for superclass invocation here; it seemed safe to call through since (I believe) the span ignores extra `finish()` calls, but since the superclass finish() isn't (currently) doing anything else I'll just remove that redundant call.

That does bring up another question I had, which is whether the fix should actually be to just have the base `TracingPlugin.finish()` pass the argument(s) through to the span `finish()`, rather than the way I did this which is specific to the graphql resolve plugin? (I don't have context on the other plugins to know if that even makes sense)

Reply:
I _think_ it would be fine to change `TracingPlugin.finish()` to pass through the argument. Might want to double check with a grep for `finish(` in the plugins directories though, just to be sure.